Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -885,6 +885,16 @@ void ProgramShaderCache::CreateHeader()
     }
   }
 
+  std::string framebuffer_fetch_string = "";
```

Comment:
Just a nitpick, but `std::string` is empty by default, so this is slightly redundant.